### ISSUE TITLE
MOT20 

### ISSUE THREAD

**Issue by sawadikasa** [daip13/LPC_MOT#11]
Hello
I would like to test in MOT20 model .
However, would the parameters of MOT20 and MOT17 be different?
I want to know the parameters such as time_thd, and distance_thd in MOT20.Thank you a lot!

**Comment by daip13** [comment]
> Hello I would like to test in MOT20 model . However, would the parameters of MOT20 and MOT17 be different? I want to know the parameters such as time_thd, and distance_thd in MOT20.Thank you a lot!

You mean the parameters in proposal generation, right? I remembered that the parameters in proposal generation should be the same for both MOT17 and MOT20.

**Comment by sawadikasa** [comment]
Thank you very much for your reply!
yes,I mean that.
In your codes:
moving_cameras = ["MOT17-05", "MOT17-06", "MOT17-07", "MOT17-10", "MOT17-11", "MOT17-12", "MOT17-13", "MOT17-14"]
        sequence_name = os.path.basename(body_pb_file).split('-')[0] + '-' + os.path.basename(body_pb_file).split('-')[1]
        if sequence_name in moving_cameras:
            args.time_thd = 10
            args.time_thd_max = 40
            args.distance_thd_2d =  50
            args.distance_thd_2d_max = 250
        else: 
            args.time_thd = 30
            args.time_thd_max = 120
            args.distance_thd_2d =  50
            args.distance_thd_2d_max = 350
you mean "else"?
Thank you a lot!

**Comment by daip13** [comment]
> Thank you very much for your reply! yes,I mean that. In your codes: moving_cameras = ["MOT17-05", "MOT17-06", "MOT17-07", "MOT17-10", "MOT17-11", "MOT17-12", "MOT17-13", "MOT17-14"] sequence_name = os.path.basename(body_pb_file).split('-')[0] + '-' + os.path.basename(body_pb_file).split('-')[1] if sequence_name in moving_cameras: args.time_thd = 10 args.time_thd_max = 40 args.distance_thd_2d = 50 args.distance_thd_2d_max = 250 else: args.time_thd = 30 args.time_thd_max = 120 args.distance_thd_2d = 50 args.distance_thd_2d_max = 350 you mean "else"? Thank you a lot!

Yes, I think so. I remembered that the range of camera movement in MOT20 is not very large. So we can use the params for static cameras.

**Comment by sawadikasa** [comment]
OK.Thank you a lot!
